Getting_Started.md

```diff
@@ -121,10 +127,12 @@ For performance reaons, you can only compile ONNX models using Lilith, python an
 Now, we use `setup` to create a proving and verifying key for our circuit, using the SRS and our compiled `.ezkl ` onnx model. 
 
 ```bash
-ezkl setup -M network.ezkl --srs-path=kzg.srs --vk-path=vk.key --pk-path=pk.key
+ezkl setup -M network.ezkl --srs-path="/Users/user/.ezkl/srs/kzg17.srs" --vk-path=vk.key --pk-path=pk.key
```

Comment:
A more robust alternative to this change, it is to update the previous srs generation command with something like this:

`ezkl get-srs --settings-path settings.json --srs-path kzg.srs`

instead of

`ezkl get-srs`

Reply:
this seems viable too